client/index.html

```diff
@@ -7,6 +7,6 @@
   <body>
     <script src="/webpack-dev-server.js"></script>
     <div id="app"></div>
-    <script src="dist/build.js"></script>
+    <script src="/dist/build.js"></script>
```

Comment:
Saw it on most places in docs and elsewhere so followed it like a convention while rewriting webpack configs.